### PR TITLE
The pathlib-style is changed on line 46-47

### LINKED ISSUE #10814: update path handling in examples/inverse/mne_cov_power.py
all of our `data_path` functions now return `pathlib.Path` objects, which allows things like the following:

https://github.com/mne-tools/mne-python/blob/37a9e5587443109515a85cdc4396af4e014ecf82/examples/datasets/brainstorm_data.py#L32-L35

Compare this to the old pattern (using `os.path.join`, often imported as `op.join`):

```py
raw_path = os.path.join(data_path, 'MEG', 'bst_raw',
                        'subj001_somatosensory_20111109_01_AUX-f.ds')
```

or the *really* old pattern (using the `+` operator on plain strings):

```py
raw_path = (data_path + '/MEG/bst_raw/' +
            'subj001_somatosensory_20111109_01_AUX-f.ds')
```

Update the paths in the file mentioned in the issue title to use the new pathlib-style, which is considered the modern, best-practice way to handle file paths.

### PR DESCRIPTION
**Pull request by HamidMandi**
closes #10814

### CODE REVIEW

**Comment by adam2392** [comment]
I would remove the filler text, rename the PR title to something a bit more informative. Nice job getting this setup.

**Comment by adam2392** [comment]
If you look at the coding conventions on the contributing guide you'll see there are various commands to help you fix the errors that are being raised by the CI pipelines.

**Comment by adam2392** [comment]
If you look at the coding conventions on the contributing guide you'll see there are various commands to help you fix the errors that are being raised by the CI pipelines.

**Comment by HamidMandi** [comment]
> If you look at the coding conventions on the contributing guide you'll see there are various commands to help you fix the errors that are being raised by the CI pipelines.

Yeah, sure. I'll do that

File: examples/inverse/mne_cov_power.py [hunk 1]
```diff
@@ -34,14 +34,17 @@
...
 # %%
 # Compute empty-room covariance
 # -----------------------------
 # First we compute an empty-room covariance, which captures noise from the
 # sensors and environment.
 
-raw_empty_room_fname = op.join(
-    data_path, 'MEG', 'sample', 'ernoise_raw.fif')
+raw_empty_room_fname = (
+    data_path/ 'MEG'/ 'sample'/ 'ernoise_raw.fif')
```
Comment (drammock) [inline examples/inverse/mne_cov_power.py:47]: Style:

operators (like `/` here) should have one space on either side. Cf.

- https://mne.tools/dev/install/contributing.html#adhere-to-standard-python-style-guidelines
- https://peps.python.org/pep-0008/#other-recommendations

Comment (HamidMandi) [reply examples/inverse/mne_cov_power.py:47]: yeah. I corrected it

**Comment by adam2392** [comment]
@HamidMandi noticed you still need to fix the CI issues. Just checking in to see if you had any troubles?

File: examples/inverse/mne_cov_power.py [hunk 2]
```diff
@@ -19,9 +19,9 @@
 #
 # License: BSD-3-Clause
 
+
```
Comment (drammock) [inline examples/inverse/mne_cov_power.py:22]:
```suggestion
```

**Pull request merged**